Rename, Update Docs and Prepare for Release (#9)
nsxiv/nsxiv: README.md

![nsxiv](https://raw.githubusercontent.com/nsxiv/nsxiv/gh-pages/img/logo.png "nsxiv")

**Neo Simple X Image Viewer**
-----------------------------

nsxiv is a fork of sxiv with the purpose of trying to make sxiv
suitable for general use, and to maintain it as sxiv is no more updated.

> It is free software so that you can use it and modify it for your needs.

Reviewer: I think this can be worded a bit better. Although I don't have any suggestions atm.

![nsxiv](https://raw.githubusercontent.com/nsxiv/nsxiv/gh-pages/img/logo.png "nsxiv")

**Neo Simple X Image Viewer**
-----------------------------

> nsxiv is a fork of sxiv with the purpose of trying to make sxiv
> suitable for general use, and to maintain it as sxiv is no more updated.
> It is free software so that you can use it and modify it for your needs.

Reviewer: ```suggestion
nsxiv is a fork of now unmaintained [sxiv](https://github.com/muennich/sxiv)
with the purpose of maintaining it and adding simple, sensible features.
nsxiv is free software licensed under GPLv2 and aims to be easy to modify and customize.
```

Please file a bug report if something does not work as documented or
expected in *this* repository, after making sure you are using the latest
release of nsxiv. Contributions are welcome, see [CONTRIBUTING.md](CONTRIBUTING.md)
for details.

Features
--------

* Basic image operations, e.g. zooming, panning, rotating
* Customizable key and mouse button mappings (in [*config.h*](config.h))
* Thumbnail mode: grid of selectable previews of all images
* Ability to cache thumbnails for fast re-loading
* Basic support for multi-frame images
* Play GIF animations

> * Display image information in status bar and in X title

Reviewer 2: ```suggestion
* Display image information in status bar
* Display image name/path in X title
```

![nsxiv](https://raw.githubusercontent.com/nsxiv/nsxiv/gh-pages/img/logo.png "nsxiv")

> **Neo (or New or Not) Simple (or Small or Suckless) X Image Viewer**

Reviewer 3: Out of curiosity, do you want to keep all these "alternative names" or to decide on one?

Author: https://github.com/nsxiv/nsxiv/discussions/47#discussioncomment-1333816

--------------------------------------------------------------------

nsxiv is a fork of now unmaintained [sxiv](https://github.com/muennich/sxiv)
with the purpose of maintaining it and adding simple, sensible features.
nsxiv is free software licensed under GPLv2 and aims to be easy to modify and customize.

Please file a bug report if something does not work as documented or
expected in *this* repository, after making sure you are using the latest
release of nsxiv. Contributions are welcome, see [CONTRIBUTING.md](CONTRIBUTING.md)
for details.

Features
--------

* Basic image operations, e.g. zooming, panning, rotating

> * Customizable key and mouse button mappings (in [*config.h*](config.h))

Reviewer: ```suggestion
* Customizable key and mouse button mappings (in *config.h*)
```

* Thumbnail mode: grid of selectable previews of all images
* Ability to cache thumbnails for fast re-loading
* Basic support for multi-frame images
* Play GIF animations
* Display image information in status bar
* Display image name/path in X title

Screenshots
-----------

> **Image mode:**

Reviewer: ```suggestion
**Image mode: (Default colors)**
```

![Image](https://raw.githubusercontent.com/nsxiv/nsxiv/gh-pages/img/image.png "Image mode")

> **Thumbnail mode and different configurations:**

Reviewer: ```suggestion
**Thumbnail mode: (Custom colors)**
```

![nsxiv](https://raw.githubusercontent.com/nsxiv/nsxiv/gh-pages/img/logo.png "nsxiv")

**Neo Simple X Image Viewer**

The sole purpose of nsxiv is to be the perfect image viewer for me. It is free
software so that you can use it and modify it for your needs. Please file a bug
report if something does not work as documented or expected. Contributions are

> welcome but there is no guarantee that they will be incorporated.

Reviewer: We need to change this.
